lib/checkout.rb

```diff
@@ -0,0 +1,6 @@
+module Checkout
```

Comment:
Nice use of a module, though perhaps unnecessary at this stage as it's used in only one class?